### PR TITLE
Let a github action build all images

### PR DESCRIPTION
**Pull request by SMillerDev**
This excludes scratch and travis but can be easily extended. It also allows for an easy extension to upload all images to github afterwards

### CODE REVIEW

**Comment by issyl0** [comment]
If you want this to run on this PR (which would be good for seeing it working!), you'll have to close this and push your branch to this repo directly (not a fork).

**Comment by SMillerDev** [comment]
Maybe I should just make PRs the trigger? And make a new one later to publish on push

**Comment by SMillerDev** [comment]
Half of the builds are failing but at least they're checked now.

**Comment by sjackman** [comment]
The CentOS builds are failing with…

```
 ---> 0f25a6b1e6cf
152
Step 5/9 : RUN localedef -i en_US -f UTF-8 en_US.UTF-8 	&& useradd -m -s /bin/bash linuxbrew 	&& echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
153
 ---> Running in 5deb410f24b5
154
[error] character map file `UTF-8' not found: No such file or directory
155
[error] default character map file `ANSI_X3.4-1968' not found: No such file or directory
156
The command '/bin/sh -c localedef -i en_US -f UTF-8 en_US.UTF-8 	&& useradd -m -s /bin/bash linuxbrew 	&& echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers' returned a non-zero code: 4
157

158
##[error]Process completed with exit code 4.
```
https://github.com/Linuxbrew/docker/pull/66/checks?check_run_id=377063920
Do you have the time and interest to look into fixing this error?

**Comment by sjackman** [comment]
Nice work, Sean! Most excellent.

File: README.md [hunk 1]
```diff
@@ -11,22 +11,22 @@ docker run -it linuxbrew/alpine
...
 ## CentOS 5
 ```sh
-docker build -t linuxbrew-centos5 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos5/Dockerfile
+docker build -t linuxbrew/centos/5 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos/5/Dockerfile
 ```
 
 ## CentOS 6
 ```sh
-docker build -t linuxbrew-centos6 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos6/Dockerfile
+docker build -t linuxbrew/centos/6 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos/6/Dockerfile
```
Comment (sjackman) [inline README.md:19]: Is `linuxbrew/centos/6` a legal tag name? I've never seen more than one `/` separating the user and image name. I'd suggest `homebrew/brew-centos6`.

File: README.md [hunk 2]
```diff
@@ -51,12 +51,12 @@ docker run -it linuxbrew/travis
 
 ## Ubuntu 12.04 LTS (Precise Pangolin)
 ```sh
-docker build -t linuxbrew-precise https://raw.githubusercontent.com/Linuxbrew/docker/master/precise/Dockerfile
+docker build -t linuxbrew/ubuntu/precise https://raw.githubusercontent.com/Linuxbrew/docker/master/ubuntu/precise/Dockerfile
```
Comment (sjackman) [inline README.md:54]: I'd suggest `homebrew/brew-ubuntu12`.

File: README.md (hunk 1 above)
Comment (SMillerDev) [reply README.md:19]: It's perfectly valid (and makes it easier to find imho)

Comment (sjackman) [reply README.md:19]: News to me. It's not really typical though. I've never seen it before. I'd stick to non-slash characters in Docker images.

**Comment by issyl0** [comment]
If this is still being worked on, it needs a rebase now. 😬

**Pull request closed without merging**